### ISSUE TITLE
Not all b-lora can take effect

### ISSUE THREAD

**Issue by yatoubusha** [liusida/ComfyUI-B-LoRA#7]
I test B-LoRA-watercolor [v17], B-LoRA-drawing3[v28],B-LoRA-pen_sketch [v30], only B-LoRA-pen_sketch [v30] can reference the style. i used the workflow you provided.
![image](https://github.com/liusida/ComfyUI-B-LoRA/assets/34413340/d35008c8-6ea2-4480-be2f-79c801f69649)
![image](https://github.com/liusida/ComfyUI-B-LoRA/assets/34413340/53475ed4-c044-4a6c-8265-7502388f70eb)
![image](https://github.com/liusida/ComfyUI-B-LoRA/assets/34413340/8228b6a7-6bf7-4d1a-88a6-ad5945d5f9c6)

**Comment by liusida** [comment]
You are right. I've tried v17, the watercolor, and it is true that the dog is not in the style of watercolor. I think the watercolor style information might not be stored in these layers.

I've compared w/ and w/o the B-LoRA model, and here is the result:

![image](https://github.com/liusida/ComfyUI-B-LoRA/assets/14249458/09637aa7-8118-4c0e-9a37-d9bbcc266847)

**Comment by liusida** [comment]
![image](https://github.com/liusida/ComfyUI-B-LoRA/assets/14249458/b20b7ea8-3a18-48a1-b813-934d966acd9e)

Since this is the training image, I guess that the B-LoRA model pays too much attention to the white background. (In the previous result, images generated w/ B-LoRA are more likely to have white background.) And maybe there're other explanations/effects as well.

**Comment by yatoubusha** [comment]
but in huggingface demo, it works, do you try that?

**Comment by liusida** [comment]
> but in huggingface demo, it works, do you try that?

Yes, I have. That demo works. 

But I don't know what are the settings and parameters behind that demo. Let me ping the original author @yardenfren1996 to see if we did anything wrong.


Hi @yardenfren1996,

@yatoubusha has found that the demo on huggingface works great but we can't reproduce the result in ComfyUI. Could you show us the settings and parameters used in the huggingface demo? Thanks!

**Comment by liusida** [comment]
I've dived a bit into ComfyUI's implementation, and I've noticed that ComfyUI often implements its own code for inference.

I'm concerned that ComfyUI might not reproduce the exact results if the original inference code does not use ComfyUI, such as when using PyTorch or diffusers.
